tests/testthat/test-dotdotdot.R

```diff
@@ -0,0 +1,56 @@
+test_that("unpack_list", {
+  # unpacks a single list
+  expect_identical(
+    unpack_list(list(1, 2, 3)),
+    unpack_list(1, 2, 3)
+  )
+
+  # ... not if more args
+  expect_failure(expect_equal(
+    unpack_list(list(1, 2, 3), "a"),
+    unpack_list(1, 2, 3, "a")
+  ))
```

Comment:
My understanding is that such usage is not common and should not be done as it is confusing with the number of tests displayed in the UI.